packages/nimble-components/docs/component-status.mdx

```diff
@@ -0,0 +1,16 @@
+import { Story, Meta } from '@storybook/addon-docs';
+import { componentStatus } from './component-status.stories';
+
+<Meta title="Components/Status Table" />
```

Comment:
How do you feel about moving it next to Getting Started in the TOC.
Could kinda imagine someone thinking a "Status Table" is type of component with the current layout